src/win32.c

```diff
@@ -284,7 +284,7 @@ INT CALLBACK BrowseCallbackProc(HWND hwnd, UINT uMsg, LPARAM lp, LPARAM pData)
 gchar *win32_show_folder_dialog(GtkWidget *parent, const gchar *title, const gchar *initial_dir)
 {
 	BROWSEINFOW bi;
-	LPCITEMIDLIST pidl;
+	LPITEMIDLIST pidl;
```

Comment:
Hum… can't figure out what the heck this should be from the docs, so I ask here: why change this (and what are those?)
The docs suggests it returns a `PIDLIST_ABSOLUTE`, which doesn't really help…